app/javascript/react/components/superhero/Superheroes.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from 'react'
+
+
+const Superheroes = () => {
+
+  // placeholder search bar(for API search)
+
+  // form to create a superhero.
+    //POST to the backend(superheroesTable)
+    //form required to have same API information(name, full_name, affiliation... etc)
+
+  // use CarrierWave when creating new Superhero to review.
+  // list superheroes that have been reviewed
+
+
+  return(
+    <>
+    <h1>Search results list</h1>
+    </>
+  )
+}
+
+
+export default Superheroes;
```

Comment:
Nope. This is an unfinished feature, remove until you designate an actual feature story for this. We will not merge in unfinished feature code :/